trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_bemt_induced_velocity.py

```diff
@@ -0,0 +1,89 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# compute_bemt_induced_velocity.py
+# 
+# Created:  Jun 2021, R. Erhard 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports
+import numpy as np 
+from scipy.interpolate import interp1d
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift 
+def compute_bemt_induced_velocity(prop,geometry,cpts):  
+    """ This computes the velocity induced by the BEMT wake
+    on lifting surface control points
+
+    Assumptions:  
+       The wake contracts following formulation by McCormick.
+    
+    Source:   
+       Contraction factor: McCormick 1969, Aerodynamics of V/STOL Flight
+       
+    Inputs: 
+       prop        - propeller or rotor data structure             [Unitless] 
+       geometry    - SUAVE vehicle                                 [Unitless] 
+       cpts        - control points in segment                     [Unitless]
+    Properties Used:
+       N/A
+    """    
+    # extract parameters
+    VD = geometry.vortex_distribution
+    R  = prop.tip_radius
+    
+    # contraction factor by McCormick
+    s  = geometry.wings.main_wing.origin[0][0] - prop.origin[0][0]
```

Comment:
Don't hard code looking for the name main_wing. There's many alternative options. Here's just a few:
A) Select the largest wing
B) Do isinstance(wing,main_wing)
C) Require the user to select a wing that corresponds with the prop (think vahana config)

Reply:
Good catch, addressed using a mix of methods B and C for now